### DISCUSSION TITLE
Floating toolbar shelf (overflow menu) does not autoclose on click of any of the menus in it

### DISCUSSION THREAD

**Discussion by pmestry25** [tinymce/tinymce#9388] [General]
## 📝 Steps to reproduce
1. use toolbar_mode as floating while initializing TinyMCE.
2. Click on **toolbar overflow icon**.
3. Now, Click on any of the toolbar menus from the newly opened floating toolbar shelf (overflow menu).

### ✔️ Expected result
The floating toolbar shelf should autoclose as soon as the user clicks on any of the toolbar menus.

### ❌ Actual result
The floating toolbar shelf is not autoclosing. The user needs to again click on the overflow menu to close it as it overlaps the text written inside the container which makes it harder to read.

## 📃 Other details
- Browser: All browsers
- OS: Mac/Windows/Linux

**Comment by lostkeys** [comment]
Hi @pmestry25. When we designed and tested the more drawer, users often wanted to perform multiple actions and having the more drawer close on each action was tedious, so the decision was made to keep it open and have the user close it when they were done.

Since this isn't a bug, and more of a discussion on how TinyMCE should behave, I'll convert this to a discussion and let more people chime with their opinions.

**Comment by shayned35** [comment]
@lostkeys I respect the decision to keep it open, however at the very least it would be great to make that behaviour configurable via a flag or something so that we can control the behaviour as needed.
